libraries/Wire/Wire.cpp

```diff
@@ -477,8 +477,8 @@ uint8_t TwoWire::read_from(uint8_t address, uint8_t* data, uint8_t length, unsig
         err = m_read(&m_i2c_ctrl,data,length,!sendStop);
       }
     }
-    timeout_ms = millis() + timeout_ms;
-    while(millis() < timeout_ms && bus_status == WIRE_STATUS_UNSET && err == FSP_SUCCESS) {
+    uint32_t startMillis = millis();
```

Comment:
```suggestion
    uint32_t const start = millis();
```
Use const correctness ;)